skins/oasis/css/core/ads.scss

```diff
@@ -63,7 +63,7 @@ $taboola-small-breakpoint: "#{$screen} and (max-width: 420px)";
 		width: 100%;
 	}
 
-	#HOME_TOP_LEADERBOARD, #TOP_LEADERBOARD, #CORP_TOP_LEADERBOARD, #HUB_TOP_LEADERBOARD {
+	#HOME_TOP_LEADERBOARD, #TOP_LEADERBOARD, #CORP_TOP_LEADERBOARD, #HUB_TOP_LEADERBOARD, .tmpHeader {
```

Comment:
how about `virtual-slot`? from what I understand you use this class to copy the generated styles for the slot and then you remove this class, am I right?